manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
+PackageVersion: "12.10"
+PackageLocale: en-US
+Publisher: WatchGuard
+PublisherUrl: https://www.watchguard.com/
+PublisherSupportUrl: https://www.watchguard.com/support
+PrivacyUrl: https://www.watchguard.com/privacy
+PackageName: Mobile VPN with SSL client
+License: WatchGuard API License
+LicenseUrl: https://www.watchguard.com/wgrd-trust-center/watchguard-apis-license-agreement
+Copyright: Copyright (c) WatchGuard Technologies, Inc.
+CopyrightUrl: https://www.watchguard.com/
+ShortDescription: Mobile VPN with SSL client Setup
+Moniker: MobileVPN

```

```suggestion
Moniker: mobilevpn
```
Moniker should be written in lowercase